### PR TITLE
Simplify setup of localdev

### PR DESCRIPTION
**Pull request by butlerx**
remove testdata
remove remove need for docker-entrypoints

### CODE REVIEW

File: cd-db/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,7 @@
-FROM postgres:alpine
-MAINTAINER butlerx <butlerx@notthe.cloud>
-
-ADD setup_zen.sh   /docker-entrypoint-initdb.d/10-setup_zen.sh
+FROM postgres:9-alpine
+LABEL maintainer="butlerx <cian@coderdojo.org>"
+COPY dumps /db
```
Comment (Wardormeur) [inline cd-db/Dockerfile]: Why using a copy over a volume? 
Copy means you'll cp a 2.5G file in your image, while a volume is a symlink-link ?

File: cd-db/anon_db.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  address = '123 Fake St., Over There Ave., End of the Road',
+  phone = translate(phone, '0123456789','9999999999');
+UPDATE cd_profiles SET avatar = NULL, parent_invites = NULL, ninja_invites = NULL, twitter = NULL, linkedin = NULL;
+UPDATE sys_login SET email = 'testmail+user' || sys_login.user || '@example.com', token = NULL;
+UPDATE sys_login SET nick = email;
+UPDATE sys_reset SET nick = 'testmail+user' || sys_reset.user || '@example.com';
+UPDATE cd_agreements SET full_name='Namey McNameFace', ip_address='127.0.0.1';
+
+\c cp-dojos-development
+UPDATE cd_dojoleads SET application = jsonb_set(application::jsonb, '{championDetails, email}'::text[], ('"testmail+user' || user_id || '@example.com"')::jsonb);
```
Comment (Wardormeur) [inline cd-db/anon_db.sql]: ChampionDetails doesn't exists anymore
You need  "champion", and "venue" at least

File: cd-db/restore_db.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+set -e
+
```
Comment (Wardormeur) [inline cd-db/restore_db.sh]: Add comment about usage : drop tar.gz in /db that will be copied/mounted

File: docker-compose.yml [hunk 4]
```diff
@@ -31,152 +32,113 @@ services:
...
   frontend:
-    image: coderdojo/cp-zen-frontend
+    image: node:carbon
+    command: yarn dev
+    working_dir: /usr/src/app
+    environment:
+      - NODE_ENV=development
     volumes:
-      - ./workspace-zen/cp-translations:/usr/src/cp-translations:Z
+      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:Z
```
Comment (Wardormeur) [inline docker-compose.yml]: isn't that hte role of the image to ln -s over to node_module?
https://github.com/CoderDojo/cp-zen-frontend/blob/master/docker-entrypoint.sh#L11

File: docker-compose.yml [hunk 5]
```diff
@@ -31,152 +32,113 @@ services:
...
       - POSTGRES_HOST=db
-      - ZENTEST=false
+      - KUE_REQUIRED='true'
+      - KUE_HOST=kue
     depends_on:
       - db
+      - kue
     volumes:
-      - ./workspace-zen/cp-translations:/usr/src/cp-translations:Z
+      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:Z
```
Comment (Wardormeur) [inline docker-compose.yml]: same as upper ^

File: cd-db/Dockerfile (hunk 1 above)
Comment (butlerx) [reply cd-db/Dockerfile]: had issues with permissions of the dumps that why the next line chownes them

File: docker-compose.yml (hunk 4 above)
Comment (butlerx) [reply docker-compose.yml]: this replaces that and works more reliably than a symlink

File: update_repos.sh [hunk 6]
```diff
@@ -1,16 +1,17 @@
...
+  "cp-events-service"
+  "cp-organisations-service"
+  "cp-translations"
+  "cp-users-service"
+  "cp-zen-frontend"
+  "cp-zen-platform")
 for repo in "${repos[@]}"; do
   cd $folder/"$repo" || (echo "Couldn't access $folder/$repo" && exit)
-  git pull
+  git fetch origin
```
Comment (Wardormeur) [inline update_repos.sh]: fetch doesn't update the reset the current branch, it's fairly different from pull

Comment (butlerx) [reply update_repos.sh]: yes but if you do a pull in a forloop you could end up with a lot of merge conflicts

File: docker-compose.yml (hunk 4 above)
Comment (Wardormeur) [reply docker-compose.yml]: then update create yourself a ticket to replace the one in the repos, elsewhat both will be done

File: cd-db/anon_db.sql (hunk 2 above)
Comment (butlerx) [reply cd-db/anon_db.sql]: so what those the data look like?

File: docker-compose.yml (hunk 4 above)
Comment (butlerx) [reply docker-compose.yml]: no this doesnt run the ln script
i was goign to remove the ln scripts once this was merged

File: cd-db/anon_db.sql (hunk 2 above)
Comment (Wardormeur) [reply cd-db/anon_db.sql]: Will commit to fix, it's easier than dumping a json schema

**Pull request merged**